src/smart-components/workspaces/WorkspaceListTable.tsx

```diff
@@ -83,6 +130,12 @@ const WorkspaceListTable = () => {
               onSelect={handleBulkSelect}
             />
           }
+          clearAllFilters={clearAllFilters}
+          filters={
+            <DataViewFilters onChange={(_e, values) => onSetFilters(values)} values={filters}>
```

Comment:
I don't think `DataViewFilters` is necessary since we just have the single filter. Just using `DataViewTextFilter` is fine.

Reply:
Will remove. This was a remnant of needing multiple filters.